### PR TITLE
Honk

### PR DESCRIPTION
**Pull request by MONKEycHIMP**
# Validate [HONK](https://solscan.io/token/3ag1Mj9AKz9FAkCQ6gAEhpLSX8B2pUbPdkb9iBsDLZNB)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/Honk_Solana/status/1746091270509105165?s=20
- Coingecko/ CMC URL (If available):https://www.coingecko.com/en/coins/honk

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [x] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
- [x] Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

### CODE REVIEW

File: validated-tokens.csv [hunk 1]
```diff
@@ -1074,3 +1074,6 @@ Spurdo Spärde,spurdo,8HJ81sHVsmJMzm6XBfTgywWQXwQTmVdmXaQHm7htGPH2,9,https://baf
 LessFnGas,LFG,LFG1ezantSY2LPX8jRz2qa31pPEhpwN9msFDzZw4T9Q,7,https://mogos7pjgqvbf3z7kzixpv67uxngkrifdfoddttz2tu6ywphk33q.arweave.net/Y4zpfek0KhLvP1ZRd9ffpdplRQUZXDHOedTp7FnnVvc?ext=png, true 
 Yaku,YAKU,NGK3iHqqQkyRZUj4uhJDQqEyKKcZ7mdawWpqwMffM3s,9,https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/EK58dp4mxsKwwuySWQW826i3fwcvUK69jPph22VUcd2H/logo.png,true
 Andrea Von Speed,VONSPEED,DVzrCErBzydh92bBzSJX1dKwVvb4omwhrvNz8CwRyxxV,6,https://i.ibb.co/bmZ6pZP/8338-C1-BC-0745-4201-BE73-D39-ADCCE129-F.jpg,true
+Honk,HONK,3ag1Mj9AKz9FAkCQ6gAEhpLSX8B2pUbPdkb9iBsDLZNB,9,https://bafkreia5nmyxbrxshpiebsakviidhkfuoevpcyzonwov7bmaovce5ekzym.ipfs.nftstorage.link,true
```
Comment (MONKEycHIMP) [inline validated-tokens.csv:1077]: HONK is listed on CG & trading on Jupiter under ALL : 
CG : https://www.coingecko.com/en/coins/honk
JUPITER :  https://jup.ag/swap/SOL-HONK_3ag1Mj9AKz9FAkCQ6gAEhpLSX8B2pUbPdkb9iBsDLZNB

**Review by MONKEycHIMP — COMMENTED** [review]
Validate Honk

**Review by MONKEycHIMP — COMMENTED** [review]
Validate HONK

**Comment by MONKEycHIMP** [comment]
https://twitter.com/Honk_Solana/status/1747831167859278157?t=pt1VHN5LGvImik28VqY2rg&s=19

**Review by 0xSoju2 — CHANGES_REQUESTED** [review]
Declined because this PR screws over other tokens + adds 2 tokens.

**Pull request closed without merging**